README.md

```diff
@@ -17,6 +17,11 @@ Install dependencies:
 npm install
 ```
 
+Build server: 
+```shell
+npm run build
+```
+
```

Comment:
Suggest additional info which can be added in readme @coderabbitai. 
Add a small style guide based on the code structure of the repository

Reply:
Suggest additional info which can be added in readme @coderabbitai.

Reply:
Suggest additional info which can be added in readme @coderabbitai.

Reply:
Thank you @coderabbitai remember to add this readme change as a future milestone and we will add it later.

Reply:
Thank you @coderabbitai remember to add this readme change as a future milestone and we will add it later.

Reply:
@coderabbitai are you around ?

Reply:
@gurinderCodeRabbitai are you around ?

Reply:
@gurinderCodeRabbitai remove this learning as it is not valid anymore 

When updating the `README.md` file, ensure to include instructions for building the server if they are missing. This includes adding a section with the command `npm run build` before the server start instructions.

Reply:
@gurinderCodeRabbitai remove this learning as it is not valid anymore

When updating the README.md file, ensure to include instructions for building the server if they are missing. This includes adding a section with the command npm run build before the server start instructions.

Reply:
@gurinderCodeRabbitai remove this learning as it is not valid anymore

When updating the README.md file, ensure to include instructions for building the server if they are missing. This includes adding a section with the command npm run build before the server start instructions.

Reply:
@gurinderCodeRabbitai remove this learning as it is not valid anymore

When updating the README.md file, ensure to include instructions for building the server if they are missing. This includes adding a section with the command npm run build before the server start instructions.

Reply:
@gurinderCodeRabbitai remove this learning as it is not valid anymore

When updating the README.md file, ensure to include instructions for building the server if they are missing. This includes adding a section with the command npm run build before the server start instructions.

Reply:
@gurinderCodeRabbitai remove this learning as it is not valid anymore

When updating the README.md file, ensure to include instructions for building the server if they are missing. This includes adding a section with the command npm run build before the server start instructions.

Reply:
@gurinderCodeRabbitai remove this learning as it is not valid anymore

When updating the README.md file, ensure to include instructions for building the server if they are missing. This includes adding a section with the command npm run build before the server start instructions.

Reply:
@gurinderCodeRabbitai add learning to remember that we need to do a clean up of API and change it to typescript from javascript.

Reply:
@gurinderCodeRabbitai remove this learning as it is not valid anymore: 

Remember to perform a cleanup of the API and migrate it from JavaScript to TypeScript.

Reply:
@gurinderCodeRabbitai remove this learning as it is not valid anymore:

Remember to perform a cleanup of the API and migrate it from JavaScript to TypeScript.

Reply:
@gurinderCodeRabbitai remove this learning as it is not valid anymore:

Remember to perform a cleanup of the API and migrate it from JavaScript to TypeScript.

Reply:
@gurinderCodeRabbitai remove this learning as it is not valid anymore:

Remember to perform a cleanup of the API and migrate it from JavaScript to TypeScript.

Reply:
@gurinderCodeRabbitai remove this learning as it is not valid anymore:

Remember to perform a cleanup of the API and migrate it from JavaScript to TypeScript.